pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1731,29 +1734,39 @@ func reserveAdapters(ctx *domainContext, config types.DomainConfig) error {
 					log.Warnf("UsedByUUID %s but no status",
 						ibp.UsedByUUID)
 					extraStr = "(which is missing)"
-				} else if other.State == types.HALTING {
-					extraStr = "(which is halting)"
+				} else {
+					if other.State == types.HALTING {
+						extraStr = "(which is halting)"
+					}
+					description.ErrorEntities = []*types.ErrorEntity{{EntityID: ibp.UsedByUUID.String(), EntityType: types.ErrorEntityAppInstance}}
+					description.ErrorRetryCondition = fmt.Sprintf("Will wait for adapter to release from app: %s", other.DisplayName)
+					// XXX we have no periodic retry
+					description.ErrorSeverity = types.ErrorSeverityError
```

Comment:
I think this should be info if halting and warning if not.

Reply:
Done